R/edge_rank_plot.R

```diff
@@ -3,7 +3,7 @@ NULL
 
 # Declarations used in package check
 globalVariables(
-  names = c('edges'),
+  names = c("molecules"),
```

Comment:
Nit picking here, but I think it's best to stick to either single quotes or double quotes in the globalVariables.

Reply:
Good point!